text/0098-metric-instruments-explained.md

```diff
@@ -0,0 +1,244 @@
+# Explain the metric instruments
+
+Propose and explain final names for the standard metric instruments theorized in [OTEP 88][otep-88] and address related confusion.
+
+[otep-88]: https://github.com/open-telemetry/oteps/blob/master/text/0088-metric-instrument-optional-refinements.md
```

Comment:
nit: I usually put this at the end of the file so I can easily find all defined links.